libs/domains/services/feature/src/lib/service-details/service-details.tsx

```diff
@@ -0,0 +1,237 @@
+import { type ComponentPropsWithoutRef } from 'react'
+import { match } from 'ts-pattern'
+import { IconEnum, ServiceTypeEnum } from '@qovery/shared/enums'
+import {
+  Badge,
+  DescriptionDetails as Dd,
+  DescriptionListRoot as Dl,
+  DescriptionTerm as Dt,
+  Icon,
+  IconAwesomeEnum,
+  Link,
+  Section,
+  Tooltip,
+  Truncate,
+} from '@qovery/shared/ui'
+import { dateFullFormat, timeAgo } from '@qovery/shared/util-dates'
+import { formatBytes, formatCronExpression, twMerge } from '@qovery/shared/util-js'
+import { useService } from '../hooks/use-service/use-service'
+import { LastCommit } from '../last-commit/last-commit'
+import { ServiceDetailsSkeleton } from './service-details-skeleton'
+
+export interface ServiceDetailsProps extends ComponentPropsWithoutRef<'div'> {
+  environmentId: string
+  serviceId: string
+}
+
+export function ServiceDetails({ className, environmentId, serviceId, ...props }: ServiceDetailsProps) {
```

Comment:
Not sure about this name, but maybe `ServiceSidebarDetails` or something like that?

I think `ServicesDetails` is too general

Reply:
Ahah I think the opposite :P
UI component and Product component are 2 different things and should not be tied together.
This component name is general on purpose so we can put it in a sidebar, in main content page or whatever.
What matter is what functionality it brings from a product perspective not the container.

Reply:
Yes I know this point. If tomorrow we have other service details, not in the same place, but before the metric, for example, we will have to rename this component, no?